api/test/utils.test.ts

```diff
@@ -66,78 +66,189 @@ const exceptionalClosedDays = [
     startDateTime: "00:00",
     endDateTime: "00:00",
   },
+  {
+    // this is during BST so 2024-07-28T23:00:00.000Z is 2024-07-29T00:00:00.BST
+    overrideDate: "2024-07-28T23:00:00.000Z",
+    type: "Summer Bank Holiday",
+    startDateTime: "00:00",
+    endDateTime: "00:00",
+  },
 ];
 
 const mockDateNow = (dateToMock: string) => {
   jest.useFakeTimers().setSystemTime(new Date(dateToMock));
 };
 
 describe("getNextOpeningDates", () => {
-  it("start the dateList from today", () => {
-    mockDateNow("2024-03-12T08:00:00.000Z");
+  describe("when London is in UTC", () => {
```

Comment:
Do we need to mock the timezone to UTC to simulate the deployed lambda environment? 

See the [AWS docs for the unreserved lambda env var `TZ`](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html).

Something like https://www.npmjs.com/package/timezone-mock might help?

Reply:
It somewhat baffles me, but I don't think we do.
The problem was not **where** the code runs, because wherever you call `new Date()` it returns a UTC timestamp which is the same everywhere for a specific point in time
The issue was trying to infer the **day of the week** from that UTC timestamp. Depending on where you are, the same exact point in time can be Saturday or Sunday, and that's addressed by
```
{ weekday: "long", timeZone: "Europe/London" }
```
ie. give me the day of the week for the point in time, **in London**

I actually installed `timezone-mock` and added 
```
beforeAll(() => {
  timezone_mock.register("Australia/Adelaide")
});

afterAll(() => {
  timezone_mock.unregister()
});
```
to the test file. Tests pass whether I set the timezone to UTC, London or even Adelaide